### ISSUE TITLE
[Bug] Header and footer of QModalLayout not rendered correctly

### ISSUE THREAD

**Issue by slowaways** [quasarframework/quasar#1184]
QModalLayout header and footer not rendered.
I've compiled and recompiled several times and sometimes it compiles right, sometimes it does not.
No error is displayed on the terminal or console.

### Software version

Quasar: 0.14.7
OS: Linux Mint 18.2 (KDE Version)
Node: 6.12.0
NPM: 3.10.10
Browsers: Opera 49.0.2725.34 (Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.62 Safari/537.36 OPR/49.0.2725.34)
iOS: -
Android: 7.1.1 (Tested with Quasar Play and same problem)

### What did you get as the error?
Nothing.

### What were you expecting?
- Render header (at top) and footer (at bottom):
![screenshot_20171125_155955](https://user-images.githubusercontent.com/9668277/33233836-91f28f8a-d1fb-11e7-9e8e-ad09bc7cae3c.png)

### What happened outside the expected?
- The header and footer position is out of order:
![screenshot_20171125_155931](https://user-images.githubusercontent.com/9668277/33233846-c9205ab4-d1fb-11e7-8800-f7f1c4d01c4e.png)

### What steps did you take, to get the error?
```
...
    <q-btn flat @click="$refs.addAlert.open()"><q-icon name="alarm_add" /></q-btn>

    <q-modal ref="addAlert" :content-css="{minWidth: '80vw', minHeight: '80vh'}">
      <q-modal-layout>
        <q-toolbar slot="header">
          <div class="q-toolbar-title">
            Header
          </div>
          <q-btn flat @click="$refs.addAlert.close()">
            <q-icon name="close" />
          </q-btn>
        </q-toolbar>
        <q-toolbar slot="footer">
          <div class="q-toolbar-title">
            Footer
          </div>
        </q-toolbar>
        <div class="layout-padding">
          <h1>Modal</h1>
          <p class="caption" v-for="n in 15">This is a Modal presenting a Layout.</p>
        </div>
      </q-modal-layout>
    </q-modal>
...
```
`quasar dev --play`

**Comment by pdanpdan** [comment]
It has something to do with vue after 2.5.3. please use 2.5.3 until it is fixed.

**Comment by slowaways** [comment]
Thanks @pdanpdan!
>= 2.5.3 bug.
<= 2.5.2 works fine!

**Comment by rstoenescu** [comment]
This is fixed in future v0.15
